tests/test_feedexport.py

```diff
@@ -1163,3 +1180,371 @@ def test_multiple_feeds_failing_logs_blocking_feed_storage(self):
         print(log)
         for fmt in ['json', 'xml', 'csv']:
             self.assertIn('Error storing %s feed (2 items)' % fmt, str(log))
+
+
+class BatchDeliveriesTest(FeedExportTestBase):
+    __test__ = True
+    _file_mark = '_%(batch_time)s_#%(batch_id)02d_'
+
+    @defer.inlineCallbacks
+    def run_and_export(self, spider_cls, settings):
+        """ Run spider with specified settings; return exported data. """
+
+        FEEDS = settings.get('FEEDS') or {}
+        settings['FEEDS'] = {
+            urljoin('file:', file_path): feed
+            for file_path, feed in FEEDS.items()
+        }
+        content = defaultdict(list)
+        try:
+            with MockServer() as s:
+                runner = CrawlerRunner(Settings(settings))
+                spider_cls.start_urls = [s.url('/')]
+                yield runner.crawl(spider_cls)
+
+            for path, feed in FEEDS.items():
+                dir_name = os.path.dirname(path)
+                for file in sorted(os.listdir(dir_name)):
```

Comment:
It looks like tests fail here on Windows: https://scrapinghub.visualstudio.com/scrapy/_build/results?buildId=98&view=logs&j=76ed1102-9071-5354-720f-6f09bbc9f0c0&t=dc276db5-d83b-509a-4a8d-770723557532&l=2129